bot/commands/admin/createRoleWatch.js

```diff
@@ -0,0 +1,88 @@
+const source = require('rfr');
+
+const { sendMessage, getChannel } = source('bot/utils/util');
+
+const {
+    generateMessageContent,
+    reactToMessage,
+    checkRoles,
+} = source('bot/utils/reactionMessage');
+
+const { Role } = source('models/role');
+
+const { parseRoleMessage } = source('bot/utils/roleParsing');
+const logger = source('bot/utils/logger');
+
+function createRoleReaction(message, args, config) {
```

Comment:
We should be consistent with naming here and call our function `createRoleWatch` or rename the file `createRoleReaction.js`. However this may result in having to rename a whole bunch of things so no worries if its too much work. Or is there a reason you chose to use roleWatch instead of roleReactions in some of your files?

Reply:
ya my naming has been fairly arbitrary, I dont quite like either of the names, if you have a preference or can think of anything else Id be open to changing it to that

Reply:
Okay lets address all that stuff in a separate PR then